### ISSUE TITLE
Ensure our iframes are a11y compliant and then ignore them in pa11y

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#11269]
We've been getting a warning by Pa11y about an non-compliant `iframe` ([first Slack mention](https://wellcome.slack.com/archives/CUA669WHH/p1728488844475049)).
We don't know what it is or where it comes from, because when we go look at the page, it's never in the DOM.
It's also often in a different page. But there's always just one error per report!

Running the tests against localhost always come back clean.
So I'm thinking it's either a Puppeteer issue, a Chrome issue, or a ghost issue. 

I've tried to fix it by adding a `wait` time before running the tests, as well as adding `cookies`, thinking that maybe the cookie banner was the culprit, but no success.

Because we have a limited amount of iframes and we have control over whether or not they have titles, we've decided we should [fix the one that doesn't](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/views/components/Iframe/Iframe.tsx) (which we also don't think is the culprit because it shows on pages that don't allow the Iframe slice 🤷‍♀ ), then add the rule (`WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1`) to [pa11y's ignore list](https://github.com/pa11y/pa11y/blob/main/README.md#ignore-array) if the issue still comes up.

I don't love ignoring issues, but this feels contained and rare enough to go for it.

--------

```
 {
      "documentTitle": "In Plain Sight British Sign Language tour with subtitles | Wellcome Collection",
      "pageUrl": "https://wellcomecollection.org/guides/exhibitions/YzwsAREAAHylrxau/bsl",
      "issues": [
        {
          "code": "WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1",
          "type": "error",
          "typeCode": 1,
          "message": "Iframe element requires a non-empty title attribute that identifies the frame.",
          "context": "<iframe height=\"0\" width=\"0\" style=\"display: none; visibility: hidden;\"></iframe>",
          "selector": "html > body > iframe",
          "runner": "htmlcs",
          "runnerExtras": {}
        }
      ]
 },
````

**Comment by rcantin-w** [comment]
Merged to prod
